sbt/src/sbt-test/project/unified/build.sbt

```diff
@@ -0,0 +1,70 @@
+import Dependencies._
+import sbt.internal.CommandStrings.{ inspectBrief, inspectDetailed }
+import sbt.internal.Inspect
+
+lazy val root = (project in file("."))
+  .settings(
+    Global / cancelable := true,
+    ThisBuild / scalaVersion := "2.12.3",
+    console / scalacOptions += "-deprecation",
+    Compile / console / scalacOptions += "-Ywarn-numeric-widen",
+    projA / Compile / console / scalacOptions += "-feature",
+    Zero / Zero / name := "foo",
+
+    libraryDependencies += uTest % Test,
+    testFrameworks += new TestFramework("utest.runner.Framework"),
+
+    commands += Command("inspectCheck", inspectBrief, inspectDetailed)(Inspect.parser) {
```

Comment:
@jvican Added the inspect output test here.
Since the parser is shared, I didn't feel the need to test various modes for inspect, but the default mode has displays the keys, so I've added the test.